dbt_server/logging.py

```diff
@@ -91,19 +91,14 @@ def configure_uvicorn_access_log():
 
 
 # Push event messages to root python logger for formatting
-def log_event_to_console(event: BaseEvent):
-    logging_method = dbt_event_to_python_root_log[event.log_level()]
-    if logging_method == logging.root.debug:
-        # Only log debug level for dbt-server logs
-        return
+def log_event_to_console(event: EventMsg):
+    logging_method = dbt_event_to_python_root_log[event.info.level]
+    # if logging_method == logging.root.debug:
```

Comment:
Still OK to comment?

Reply:
Yes, this isn't really working yet anyway because of celery, and I've seen some instances where debug logs were helpful so I didn't want to filter out yet just in case. This is next on the docket for me to work on!